deployment/manifests/aws.pp

```diff
@@ -22,6 +22,10 @@
 $solr_data_set_manager_data = "${data_dir}/solr/data_set_manager"
 $solr_core_data = "${data_dir}/solr/core"
 $django_docker_engine_data_dir = "${data_dir}/django-docker-engine-data"
+$django_docker_engine_mem_limit_mb = 3584
```

Comment:
I'd drop 'mb' from name since changing the units would require changing the code everywhere this variable is used.

Reply:
The standard practice is to add a comment specifying the units. Compare:
$django_docker_engine_mem_limit_mb = 3584
vs
$django_docker_engine_mem_limit = 3584  # size in MB

See also:
https://github.com/galaxyproject/galaxy/blob/dev/config/galaxy.yml.sample#L853
https://github.com/galaxyproject/galaxy/blob/dev/config/galaxy.yml.sample#L1586

Reply:
I really don't believe there is a single universal standard here, but thank you for looking at it.